### PR TITLE
Disable permissions change

### PR DESCRIPTION
**Pull request by eduardosan**
Hi, 

I have a very specific use case where data volumes are mounted from a sshfs partition and other network partitions, and in this case one can't change permissions on remote disks.

I just added an environment var to disable this when it's necessary.

### CODE REVIEW

**Comment by BertrandGouny** [comment]
Hello,
thanks for the pull request.

Can you please adapt it to the existing formatting : 
 * use 2 spaces as indentation
 * close if statements with  `fi`

Will merge that in the 1.2.3 coming release.
Thanks

**Comment by kumy** [comment]
@eduardosan Could you please fix the formatting, so we could all benefit from your fix :)

**Comment by eduardosan** [comment]
Hello,

The fix is provided in above commit.

File: image/service/slapd/startup.sh [hunk 1]
```diff
@@ -125,7 +127,8 @@ EOF
       mv /tmp/schema/cn=config/cn=schema/* /etc/ldap/slapd.d/cn=config/cn=schema
       rm -r /tmp/schema
 
-      chown -R openldap:openldap /etc/ldap/slapd.d/cn=config/cn=schema
+      if [ -z "$DISABLE_CHOWN" ]; then
+        chown -R openldap:openldap /etc/ldap/slapd.d/cn=config/cn=schema
```
Comment (kumy) [inline image/service/slapd/startup.sh]: missing `fi`?

File: image/service/slapd/startup.sh [hunk 2]
```diff
@@ -306,10 +310,11 @@ EOF
...
       # create DHParamFile if not found
       [ -f ${LDAP_TLS_DH_PARAM_PATH} ] || openssl dhparam -out ${LDAP_TLS_DH_PARAM_PATH} 2048
-      chmod 600 ${LDAP_TLS_DH_PARAM_PATH}
-
+      
       # fix file permissions
-      chown -R openldap:openldap ${CONTAINER_SERVICE_DIR}/slapd
+      if [ -z "$DISABLE_CHOWN" ]; then
+        chmod 600 ${LDAP_TLS_DH_PARAM_PATH}
+        chown -R openldap:openldap ${CONTAINER_SERVICE_DIR}/slapd
```
Comment (kumy) [inline image/service/slapd/startup.sh]: missing `fi`?

File: image/service/slapd/startup.sh [hunk 3]
```diff
@@ -202,8 +205,9 @@ EOF
       ssl-helper $LDAP_SSL_HELPER_PREFIX $PREVIOUS_LDAP_TLS_CRT_PATH $PREVIOUS_LDAP_TLS_KEY_PATH $PREVIOUS_LDAP_TLS_CA_CRT_PATH
       [ -f ${PREVIOUS_LDAP_TLS_DH_PARAM_PATH} ] || openssl dhparam -out ${LDAP_TLS_DH_PARAM_PATH} 2048
 
-      chmod 600 ${PREVIOUS_LDAP_TLS_DH_PARAM_PATH}
-      chown openldap:openldap $PREVIOUS_LDAP_TLS_CRT_PATH $PREVIOUS_LDAP_TLS_KEY_PATH $PREVIOUS_LDAP_TLS_CA_CRT_PATH $PREVIOUS_LDAP_TLS_DH_PARAM_PATH
+      if [ -z "$DISABLE_CHOWN" ]; then
+        chmod 600 ${PREVIOUS_LDAP_TLS_DH_PARAM_PATH}
+        chown openldap:openldap $PREVIOUS_LDAP_TLS_CRT_PATH $PREVIOUS_LDAP_TLS_KEY_PATH $PREVIOUS_LDAP_TLS_CA_CRT_PATH $PREVIOUS_LDAP_TLS_DH_PARAM_PATH
```
Comment (kumy) [inline image/service/slapd/startup.sh]: indent + `fi`

**Comment by eduardosan** [comment]
Now I thing they are all included

**Comment by kumy** [comment]
@eduardosan Thanks :+1: 

@BertrandGouny Do you think it can be merged? Thanks.

**Comment by BertrandGouny** [comment]
Thanks all,
merge in next release

**Pull request merged**